KNOWN_ISSUES.md

```diff
@@ -0,0 +1,72 @@
+#  Known issues in version 12.0 alpha 2
```

Comment:
I'd keep the file around while in alpha, to quickly keep record of what's still open / not working in the respective commit. I also use(d) that list for the public alphas so far.

Alternatively, we could break those out in GitHub issues, but since there are many subtleties in there, too, I'm not sure it's worth the time.